components/menu/stories/menu.stories.js

```diff
@@ -237,13 +249,16 @@ TraySubmenu.parameters = {
 };
 
 export const MenuItem = MenuItemGroup.bind({});
+MenuItem.tags = ["!autodocs"];
 MenuItem.argTypes = {
 	isDisabled,
 	isActive,
 	isFocused,
 	isHovered,
-	isOpen,
```

Comment:
`isOpen` is used with menu items, but only for Collapsible and submenus, which we're not demonstrating in this story, it didn't feel like a good fit to keep this state in the controls